### PR TITLE
Build desktop features

### PR DESCRIPTION
**Pull request by Armynerh**
make changes to portfolio for responsiveness on large devices
add new images
change style.css to pass linters
change index.html to include desktop images

### CODE REVIEW

**Review by laiifuu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: 

## Hi @Armynerh :wave:  
You've done a nice job so far on the desktop view of your portfolio project :clap: :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!
so-close

## Highlights
- [x] Use of media queries :heavy_check_mark: 
- [x] Use of both `Flexbox` and `Grid` :heavy_check_mark: 

## Required Changes ♻️
Starting with your README: 
- [ ] Kindly replace [your project name] with the name of your project
Screenshot from 2023-06-15 14-51-44
- [ ] Kindly remove the description of the `Live Demo Link` section:
Screenshot from 2023-06-15 14-53-03

Moving on to the portfolio: 
- [ ] If you check the figma template, you'll see that there's a button with a down arrow under the social media icons, you should add it to stick as much as possible to the design.
Screenshot from 2023-06-15 14-56-41

- [ ] In the mobile view, the skills cards are not perfectly centered, there's more space on the right than on the left: 
Screenshot from 2023-06-15 15-22-57

- [ ] In the desktop view, if you check the figma template, the background image should start at the top of the page, not at the start of the hero section: 
    - The template: 
    Screenshot from 2023-06-15 15-08-36
    - Your design: 
    Screenshot from 2023-06-15 15-08-52

- [x] In the desktop view, if you check the figma template, there should be more spacing on the left and right of the following sections: `My recent works`, `About me`, `Contact me`
    - The template: 
    Template 3-main

- [x] In the desktop view, if you check the figma template, you'll see that the footer which holds the social media icons sticks at the bottom of the page with a top border: 
    - The template: 
    Screenshot from 2023-06-15 15-25-53
    - Your design: 
    Screenshot from 2023-06-15 15-26-22

_Check the comments under the review :point_down: _

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and :computer: Happy coding! :computer: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -19,20 +19,22 @@
...
                     <h2>Hey there. <span id="main-span1">I'm Kenny.</span> <span id="main-span2">I'm a software developer</span></h2>
                 </div>
                 <div id="main-item2" class="font-normal">
+                    <div id="main-p">
                     <p>
                     I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
                     don’t hesitate to contact me.
                     </p>
                 </div>
+                </div>
```
Comment (osamaashraf6) [inline index.html:37]: - [x] Kindly 😃 as stated by the previous reviewer, If you check the Figma template, you'll see that there's a button with a down arrow under the social media icons,  so please I suggest adding it to stick as much as possible to the design.💯🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
| image    |     image  |

File: index.html [hunk 2]
```diff
@@ -271,11 +282,10 @@ <h4>I'm always interested in hearing about new projects, so if you'd like to cha
...
                 </div>
-            </div>
-            <div id="shape">
-                <img src="images/shape.svg">
-            </div>
-        </section>
+                <div id="shape">
+                    <img src="images/shape.svg">
+                </div>
+        </footer>
```
Comment (osamaashraf6) [inline index.html:290]: - [x] Kindly 😃 as stated by the previous reviewer, In the desktop view, if you check the Figma template, you'll see that the footer which holds the social media icons sticks at the bottom of the page with a top border, so please try to make that to make the design nice like the original💯🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|  image   |     image |

File: style.css [hunk 3]
```diff
@@ -510,22 +509,252 @@
...
+    flex-wrap: wrap;
+    justify-content: center;
+    background-image: none;
+    text-align: center;
+  }
+
+  .main-container1 {
+    background-image: url(images/Header-llustration-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
```
Comment (osamaashraf6) [inline style.css:549]: - [x] You did a great job 😃at implementing the most changes from the previous reviewer. however I see that the previous reviewer told you to put the background image at the top of the page, not the section and you did a great effort to put it, but as stated by the previous reviewer please try to stick to the Figma Design and move the background image little to up to make it nice like the Figma design 💯🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   image  |    image   |

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @Armynerh 😊, 

Good job so far!
There are some issues that you still need to work on from the previous reviewer to go to the next project but you are almost there!
SoCloseAlmostGIF (3)

## Highlights🎉🎉

- [ ] Following Figma design😊👌
- [ ] Use the semantics elements💯
- [ ] Nice design .🚀
- [ ] Use the flexbox to place the items 😃👍

## Required Changes ♻️

- [x] Kindly 😃 as stated by the previous reviewer, Kindly replace [your project name] with the name of your project to make the Readme professional💯

| Your Readme| 
| --- |  
|  image |

- [x] Kindly 😃 as stated by the previous reviewer, Kindly remove the description of the Live Demo Link section to make the Readme professional🚀

| Your Readme| 
| --- |  
|  image |

_Check the comments under the review._

## Optional suggestions
- N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (emmiiorji) [reply style.css:549]: > * [x]  You did a great job 😃at implementing the most changes from the previous reviewer. however I see that the previous reviewer told you to put the background image at the top of the page, not the section and you did a great effort to put it, but as stated by the previous reviewer please try to stick to the Figma Design and move the background image little to up to make it nice like the Figma design 💯🚀
> 
> > Check below to see the difference 😊⤵️
> > If you can not see the image clearly just click on it to open a new tab in the browser😊
> 
> Original design	Your design
> image	image

Well done for your great efforts so far. While trying to implement the previous reviewer's suggestion, you seem to have introduced another issue where the navigation items are fixed.
Kindly close up the white space as highlighted below and make the items not to be fixed.

Your design:
image image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Armynerh, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- You have implemented most of the required changes


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @Armynerh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**